parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueWriters.java

```diff
@@ -113,8 +115,8 @@ private ParquetValueWriters() {
     return new MapWriter<>(dl, rl, keyWriter, valueWriter);
   }
 
+  @SuppressWarnings("checkstyle:VisibilityModifier")
```

Comment:
why is this suppressed?

Reply:
This was due to `column` being directly accessed by its child classes; originally I added `id` in the same way so I moved this suppression out, but now since we don't need to make change to this class, I'll move it back.